api/configuration/v1/kongconsumer_types.go

```diff
@@ -75,6 +75,9 @@ type KongConsumerSpec struct {
 	// ControlPlaneRef is a reference to a ControlPlane this Consumer is associated with.
 	// +optional
 	ControlPlaneRef *configurationv1alpha1.ControlPlaneRef `json:"controlPlaneRef,omitempty"`
+
+	// Tags is an optional set of tags applied to the consumer.
+	Tags configurationv1alpha1.Tags `json:"tags,omitempty"`
```

Comment:
The type shared by resources in all versions may need to be moved to some common package to prevent potential cyclic dependency on packages.

Reply:
Extracted `Tags` type to a `common` package.